apps/telemetry_storage/management/commands/seed_channels.py

```
@@ -0,0 +1,145 @@
+                        )
+                    )
+
+            else:
+                self.stdout.write(
+                    self.style.WARNING(
+                        f"Verification failed: Expected 1 active channel, found {active_count}"
+                    )
+                )
+
+        except Exception as e:
+            self.stderr.write(self.style.ERROR(f"Error seeding channels: {str(e)}"))
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)